frontends/precinct-scanner/src/api/hmpb.ts

```diff
@@ -82,7 +82,10 @@ export function addTemplates(pkg: BallotPackage): AddTemplatesEvents {
           );
         }
 
-        await fetch('/scan/hmpb/addTemplates', { method: 'POST', body });
+        await fetch('/precinct-scanner/config/addTemplates', {
```

Comment:
Just double checking that the renaming of this path was intentional. I assume this is one of the cases you're referring to when you say the following in the PR description:

> I also changed a couple precinct scanner API endpoints to be a bit more consistent.

If this is intentional and this better fits under `/precinct-scanner/config`, should we move it to the `src/api/config.ts` file?

Reply:
Yeah it was intentional. I was trying to restrain myself from further renaming/refactoring, but I think you're right that I left things kind of halfway changed here. I will update `src/api` to match.

The other thing I would like to do is make the casing consistent - currently we use a mix of hyphens and camelCase. But I think that could be a separate PR

Reply:
5cc9a9c6e